### PR TITLE
Suggestions for Level 3

### PR DESCRIPTION
**Pull request by ajhyndman**

### CODE REVIEW

File: js/level3.js [hunk 1]
```diff
@@ -98,15 +161,17 @@
...
-    another property for it - .innerHTML
+    Ok, so far so good. But what if we want only the text from our 'h1' tag?
+    Page elements have another property for this: '.textContent'
+
     Example:
-    ourTwitter.innerHTML;
-    //we will get text '@NodeGirlsSydney'
+
+    ourTwitter.textContent;
+    // we will get text '@NodeGirlsSydney'
```
Comment (ButenkoT) [inline js/level3.js]: `we will get text "Twitter: @NodeGirlsSydney @NodeGirlsMelb"` as we had updates in html

File: js/level3.js [hunk 2]
```diff
@@ -183,9 +269,11 @@
...
 
 
-//TODO: Well, do you still have kittens on your screen? I like both logo and kittens.
-//Let's create a new image with source logo and put it into header.
-//PS: you also can give styles to the new node that you create.
+// TODO: Well, do you still have kittens on your screen? I like both logo and
+// kittens. Let's create a new image that sources our original logo file, and
+// put it into header.
+//
+// P.S. Wou also can give styles to the new node that you create.
```
Comment (ButenkoT) [inline js/level3.js]: mistype `You also ...`

**Comment by ButenkoT** [comment]
Thank you a lot for polishing instructions! I very appreciate your help :)

**Comment by ajhyndman** [comment]
@ButenkoT I think I fixed your two suggestions.

I'm glad you found it useful.  :)

**Comment by ButenkoT** [comment]
Those changes were right in time, as we run one event tomorrow.

**Comment by ajhyndman** [comment]
Oh, I didn't even realize that.

I hope it goes great!

**Pull request merged**